### PR TITLE
feat: add support for video embed

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -6,28 +6,28 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #5A8CE9;
-  --ifm-color-primary-dark: #5A8CE9;
+  --ifm-color-primary: #5a8ce9;
```
Comment (IlyaBondar) [inline src/css/custom.css:9]: it's the same value but in lowercase. For what do we need this change?

File: docusaurus.config.js [hunk 2]
```diff
@@ -4,54 +4,71 @@
...
+import fauxOembedTransformer from '@remark-embedder/transformer-oembed';
+
+const transformer = {
+  ...fauxOembedTransformer,
+  getHTML: async (url, getConfig = {}) => {
+    let html = await fauxOembedTransformer.getHTML(url, getConfig);
+    if (html?.startsWith('<iframe')) {
+      html = html.replace('width="200"', 'class="iframe-video"');
+      html = html.replace('height="113"', '');
+    }
```
Comment (IlyaBondar) [inline docusaurus.config.js:18]: for what do we need it?

Comment (sr-remsha) [reply docusaurus.config.js:18]: we have added the support for rendering of videos

File: src/css/custom.css (hunk 1 above)
Comment (sr-remsha) [reply src/css/custom.css:9]: formatting

Comment (IlyaBondar) [reply src/css/custom.css:9]: I don't think it's auto-formatted(see the line 11) and UPPERCASE is more popular for HEX-colors

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**